yoyodyne/models/base.py

```diff
@@ -42,6 +42,7 @@ class BaseEncoderDecoder(lightning.LightningModule):
     teacher_forcing: bool
     # Decoding arguments.
     beam_width: int
+    n: int
```

Comment:
If we *are* going to keep this arg, we should at least change it to something like `beam_n` as `n` could refer to other decoding options otherwise.